web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1280,12 +1280,18 @@ $(function() {
                 } else if ('newReportAtLonlat' in e.state) {
                     fixmystreet.display.begin_report(e.state.newReportAtLonlat, false);
                 } else if ('page_change' in e.state) {
-                    $('.pagination').data('page', e.state.page_change.page)
+                    fixmystreet.markers.protocol.use_page = true;
+                    $('#show_old_reports').prop('checked', e.state.page_change.show_old_reports == 'on');
+                    $('.pagination:first').data('page', e.state.page_change.page) //;
                         .trigger('change.filters');
+                    if ( fixmystreet.page != 'reports' ) {
```

Comment:
I *think* it is safe to call this regardless of the page already being this (see how filter_change just below always calls it), maybe?